db/migrate/20220727084515_add_decidim_organization_references_to_decidim_participative_action.decidim_ludens.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# This migration comes from decidim_ludens (originally 20221229184753)
+
+class AddDecidimOrganizationReferencesToDecidimParticipativeAction < ActiveRecord::Migration[6.0]
+  def up
+    add_reference :decidim_participative_actions, :decidim_organization, null: false, foreign_key: true
+    Decidim::Ludens::ParticipativeAction.where(organization: nil)
+                                        .find_each { |pa| pa.update!(organization: Decidim::Organization.first) }
```

Comment:
Here I think we should do it for every Organization since there is multi tenant in production